client/src/components/TabularViewComponent/TabularViewComponent.js

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import React, { Component } from "react";
+import { DetailsList, SelectionMode } from "@fluentui/react/lib/DetailsList";
+import LoaderComponent from "../LoaderComponent/LoaderComponent";
+
+import "./TabularViewComponent.scss";
+
+export class TabularViewComponent extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      isLoading: true,
+      items: this.generateItems(),
+      columns: this.generateColumns()
+    };
+  }
+
+  componentDidMount = () => {
+    this.setState({ isLoading: false });
+  }
+
+  generateItems = () => {
+    const items = [];
+    this.props.relationships.forEach(element => {
+      const item = {};
+      item.name = element.$relationshipName;
+      item.source = element.$sourceId;
+      item.target = element.$targetId;
+      item.id = element.$relationshipId;
+      items.push(item);
+    });
+    return items;
+  }
+
+  generateColumns = () => [
+    {
+      key: "nameColumn",
+      fieldName: "name",
+      name: "Name",
```

Comment:
These column titles should probably be localized?

Also, can't this array just be a constant? I don't think we need to generate it every time as it doesn't change?

Reply:
Change into Translate Json  and constant